web/src/features/auth/slices/authenticationSlice.ts

```diff
@@ -86,6 +87,12 @@ export const decodeRoles = (token: string | undefined) => {
   }
 }
 
+export const testAuthenticate =
```

Comment:
Mentioned this before - but maybe a better name? My mind immediately starts expecting that I'm in a test file.